docs/source/merlin_variables.rst

```diff
@@ -31,73 +31,138 @@ The directory structure of merlin output looks like this:
 Reserved variables
 ------------------
 .. list-table:: Study variables that Merlin uses. May be referenced within a specification file, but not reassigned or overridden.
-
-  * - Variable
-    - Description
-    - Example Expansion
-  * - ``$(SPECROOT)``
-    -  Directory path of the specification file.
-    - ``/globalfs/user/merlin_workflows``
-  * - ``$(OUTPUT_PATH)``
-    - Directory path the study output will be written to. If not defined
-      will default to the current working directory. May be reassigned or
-      overridden.
-    - ``./studies``
-  * - ``$(MERLIN_TIMESTAMP)``
-    - The time a study began. May be used as a unique identifier.
-    - ``"YYYYMMDD-HHMMSS"``
-  * - ``$(MERLIN_WORKSPACE)``
-    - Output directory generated by a study at ``OUTPUT_PATH``. Ends with
-      ``MERLIN_TIMESTAMP``.
-    - ``$(OUTPUT_PATH)/ensemble_name_$(MERLIN_TIMESTAMP)``
-  * - ``$(WORKSPACE)``
-    - The workspace directory for a single step.
-    - ``$(OUTPUT_PATH)/ensemble_name_$(MERLIN_TIMESTAMP)/step_name/``
-  * - ``$(MERLIN_INFO)``
-    - Directory within ``MERLIN_WORKSPACE`` that holds the provenance specs and sample generation results.
-      Commonly used to hold ``samples.npy``.
-    - ``$(MERLIN_WORKSPACE)/merlin_info/``
-  * - ``$(MERLIN_SAMPLE_ID)``
-    - Sample index in an ensemble
-    - ``0`` ``1`` ``2`` ``3``
-  * - ``$(MERLIN_SAMPLE_PATH)``
-    - Path in the sample directory tree to a sample's directory, i.e. where the
-      task is actually run.
-    - ``/0/0/0/`` ``/0/0/1/`` ``/0/0/2/`` ``/0/0/3/``
-  * - ``$(MERLIN_GLOB_PATH)``
-    - All of the directories in a simulation tree as a glob (*) string
-    - ``/\*/\*/\*/\*``
-  * - ``$(MERLIN_PATHS_ALL)``
-    - A space delimited string of all of the paths;
-      can be used as is in bash for loop for instance with:
-
-      .. code-block:: bash
-
-         for path in $(MERLIN_PATHS_ALL)
-           do
-             ls $path
-           done
-
-         for path in $(MERLIN_PATHS_ALL)
-         do
-           ls $path
-         done
-    - ``0/0/0 0/0/1 0/0/2 0/0/3``
-  * - ``$(MERLIN_SAMPLE_VECTOR)``
-    - Vector of merlin sample values
-    - ``$(SAMPLE_COLUMN_1) $(SAMPLE_COLUMN_2) ...``
-  * - ``$(MERLIN_SAMPLE_NAMES)``
-    - Names of merlin sample values
-    - ``SAMPLE_COLUMN_1 SAMPLE_COLUMN_2 ...``
-  * - ``$(MERLIN_SPEC_ORIGINAL_TEMPLATE)``
-    - Copy of original yaml file passed to ``merlin run``.
-    - ``$(MERLIN_INFO)/*.orig.yaml``
-  * - ``$(MERLIN_SPEC_EXECUTED_RUN)``
-    - Parsed and processed yaml file with command-line variable substitutions included.
-    - ``$(MERLIN_INFO)/*.partial.yaml``
-  * - ``$(MERLIN_SPEC_ARCHIVED_COPY)``
-    - Archive version of ``MERLIN_SPEC_EXECUTED_RUN`` with all variables and paths fully resolved.
-    - ``$(MERLIN_INFO)/*.expanded.yaml``
+   :widths: 25 50 25
```

Comment:
is this working as intended?

Reply:
I added that :widths: tag prior to modifying the css for tds to have a max-width of 300px so there's a chance it's not needed anymore. Let me go check.

Reply:
It looks like that's not even needed with the new custom css. I'll remove it and push again.